integration_tests/src/main/python/orc_write_test.py

```diff
@@ -90,6 +91,31 @@ def test_part_write_round_trip(spark_tmp_path, orc_gen):
             data_path,
             conf = {'spark.rapids.sql.format.orc.write.enabled': True})
 
+
+@ignore_order(local=True)
+@pytest.mark.parametrize('orc_gen', [int_gen], ids=idfn)
+@pytest.mark.parametrize('orc_impl', ["native", "hive"])
+def test_dynamic_partition_write_round_trip(spark_tmp_path, orc_gen, orc_impl):
```

Comment:
Don't we need the equivalent test for other output formats that support dynamic partition writes?  Can be tracked as a separate issue.

Reply:
thought about adding that here, but didn't thinking it would clubber up the PR. Filed this: https://github.com/NVIDIA/spark-rapids/issues/7382